client/my-sites/site-settings/form-analytics.jsx

```diff
@@ -71,6 +72,7 @@ class GoogleAnalyticsForm extends Component {
 			jetpackVersionSupportsModule,
 			showUpgradeNudge,
 			site,
+			siteId,
```

Comment:
is it possible get the `siteId` from the current `site` param? something like `site.ID`?